fuse_optimizers/src/variable_stamp_index.cpp

```diff
@@ -58,7 +58,8 @@ ros::Time VariableStampIndex::currentStamp() const
   }
   else
   {
-    return ros::Time(0, 0);
+    // NOTE(CH3): Might be useful to figure out where to place uninitialized checks later on...
+    return rclcpp::Time(0, 0, default_clock_type);  // NOTE(CH3): Signals uninitialized
```

Comment:
`RCL_CLOCK_UNINITIALIZED` would make sense to me here if `rclcpp::Time::is_valid()` existed. Then there's no need for this parameter to get passed.

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/9a1142173a60e2d50cb7bb40180b0ff85c8f486a